libs/ui/elements/src/lib/markdown-parser/markdown-parser.component.css

```diff
@@ -1,5 +1,5 @@
 /** Body **/
-:host /deep/ .markdown-body {
+:host ::ng-deep .markdown-body {
```

Comment:
btw why do we need `ng-deep` if it's in the same component ?

Reply:
because the content generated through innerHtml will not be affected by the component stylesheet.